### PR TITLE
Update SAML ocs to be a bit more clear

### PR DESCRIPTION
**Pull request by plessbd**
Clarifying the SAML docs a bit.

### CODE REVIEW

File: docs/simpleSAMLphp.md [hunk 1]
```diff
@@ -126,7 +126,17 @@ $config = array(
 );
 ```
 
-Use the above example to match the response from the IdP, the indexes under `authproc[40]` point to indexes expected by Open XDMoD.
+Use the above example to match the response from the IdP, the indexes (left side of `=>`) under `authproc[40]` point to keys (right side of `=>`) expected by Open XDMoD.
+
+These indexes need to be what SAML receives from the IdP not the ones known from LDAP or other sources.
+
+These can be viewed by using the Test authentication sources page built into simplesamlphp located at
```
Comment (jpwhite4) [inline docs/simpleSAMLphp.md]: Perhaps put "test authentication sources" in quotes or in fixed font. Also check capitalization of the word Test.

File: docs/simpleSAMLphp.md [hunk 2]
```diff
@@ -126,7 +126,17 @@ $config = array(
...
+`https://<hostname>/simplesaml/module.php/core/authenticate.php` then clicking on `xdmod-sp` and it should redirect you to the configured IdP login page.
+After loggin in you should then be redirected back to a `SAML 2.0 SP Demo Example` page.
```
Comment (jpwhite4) [inline docs/simpleSAMLphp.md]: Is loggin a word?

File: docs/simpleSAMLphp.md [hunk 3]
```diff
@@ -126,7 +126,17 @@ $config = array(
 );
 ```
 
-Use the above example to match the response from the IdP, the indexes under `authproc[40]` point to indexes expected by Open XDMoD.
+Use the above example to match the response from the IdP, the indexes (left side of `=>`) under `authproc[40]` point to keys (right side of `=>`) expected by Open XDMoD.
+
+These indexes need to be what SAML receives from the IdP not the ones known from LDAP or other sources.
+
```
Comment (jpwhite4) [inline docs/simpleSAMLphp.md]: No new paragraph here.

File: docs/simpleSAMLphp.md (hunk 2 above)
Comment (plessbd) [reply docs/simpleSAMLphp.md]: in some places I am pretty sure we use it as loggin

File: docs/simpleSAMLphp.md (hunk 1 above)
Comment (plessbd) [reply docs/simpleSAMLphp.md]: Will add back-ticks the page title case when displayed in the browser matches the case here.

**Pull request merged**